src/components/Graph.coffee

```diff
@@ -24,13 +24,15 @@ class Graph extends noflo.Component
     if typeof graph is 'object'
       if typeof graph.addNode is 'function'
         # Existing Graph object
-        return @createNetwork graph
+        return @createNetwork graph, (err) =>
+          return @error err if err
```

Comment:
Is this and the below error handling needed here, or just added since passed over this area of code?